src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -821,24 +821,40 @@ def forward(
         return_dict: Optional[bool] = None,
     ) -> Union[Tuple, BaseModelOutputWithPooling]:
         r"""
-        Returns:
+        Return:
 
         Examples:
 
+        LayoutLMv2 depends on detectron2, torchvision and tesseract; passing doctests requires their installation. Run
+        the following to install them: `python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'`
+        `python -m pip install torchvision tesseract`
```

Comment:
This is not something the user browsing the doc will be interested to see here. It should be mentioned at the beginning of the doc page of the model (if it's not already), but that's all.

Reply:
Where is the doc page? I put it as a comment at the beginning of the file just in case that's what you are referring to.